ci/config/molecule.yaml

```diff
@@ -30,3 +30,7 @@
 - job:
     name: cifmw-molecule-cert_manager
     nodeset: centos-9-crc-xxl
+- job:
+    name: cifmw-molecule-env_op_images
+    nodeset: centos-9-crc-xl
+    timeout: 5400
```

Comment:
Can we drop this timeout now since we are no longer installing control plane in molecule tests?